### ISSUE TITLE
Create automated test of titan.sh

### ISSUE THREAD

**Issue by dalaro** [thinkaurelius/titan#566]
Use existing expect-based tooling developed to test gremlin.sh automatically during Maven's verify phase in titan-dist

**Comment by spmallette** [comment]
Could you provide a pointer to the code where you do the testing of gremlin.sh?

**Comment by dalaro** [comment]
In titan05, the code lives in these two places:
- titan-dist/src/test/java (integration tests run by failsafe)
- titan-dist/src/test/expect (expect script source)

I looked at https://github.com/cverges/expect4j early on, but I decided to go with the reference expect implementation.  I still develop expect scripts using the command-line interactive prompt, and I didn't want to have to consider potential incompatibilities between two implementations of expect.  So the Java integration tests do little more than unzip the distribution zipfile and fork an expect process, which proceeds to do most of the heavy lifting.  The expect process is just responsible for yielding an appropriate return status (0 for success or nonzero for failure).
